nebula-logger/main/logger/objects/LogEntry__c/fields/HasException__c.field-meta.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I just remembered that we might be able to do the conditional rendering in the flexipage without a new field - you could use the picklist field `LogEntry__c.Type__c == 'Exception'` to get essentially the same result. But let me know if you think it's still worth keeping the new field.

Reply:
I think we should keep it to make things consistent in terms of how the flexipage is filtered - reduces the cognitive overhead.

Reply:
Works for me!